src/material-experimental/mdc-list/list-base.ts

```diff
@@ -6,19 +6,51 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {AfterContentInit, ElementRef, NgZone, OnDestroy, QueryList} from '@angular/core';
-import {setLines} from '@angular/material/core';
+import {Platform} from '@angular/cdk/platform';
+import {
+  AfterContentInit,
+  Directive,
+  ElementRef,
+  HostBinding,
+  NgZone,
+  OnDestroy,
+  QueryList
+} from '@angular/core';
+import {RippleConfig, RippleRenderer, RippleTarget, setLines} from '@angular/material/core';
 import {Subscription} from 'rxjs';
 import {startWith} from 'rxjs/operators';
 
-export class MatListBase {}
+@Directive()
+export class MatListBase {
+  // @HostBinding is used in the class as it is expected to be extended. Since @Component decorator
+  // metadata is not inherited by child classes, instead the host binding data is defined in a way
+  // that can be inherited.
+  // tslint:disable-next-line:no-host-decorator-in-concrete
+  @HostBinding('class.mdc-list--non-interactive')
+  _isNonInteractive: boolean;
+}
 
-export class MatListItemBase implements AfterContentInit, OnDestroy {
+export class MatListItemBase implements AfterContentInit, OnDestroy, RippleTarget {
   lines: QueryList<ElementRef<Element>>;
 
+  rippleConfig: RippleConfig = {};
+
+  rippleDisabled: boolean;
+
   private _subscriptions = new Subscription();
 
-  constructor(protected _element: ElementRef, protected _ngZone: NgZone) {}
+  private _rippleRenderer: RippleRenderer;
+
+  constructor(protected _element: ElementRef, protected _ngZone: NgZone, listBase: MatListBase,
+              platform: Platform) {
+    this.rippleDisabled = listBase._isNonInteractive;
+    if (!listBase._isNonInteractive) {
+      this._element.nativeElement.classList.add('mat-mdc-list-item-interactive');
```

Comment:
Maybe move the `this_element.nativeElement` out into a variable since it's repeated in 3 places and doesn't minify very well?